simpleviewinc/re-theme
# Add svkeg repos

## Context

* Github NPM packages are a pain to work with
* They always require a git key
* They require a .npmrc to overwrite the default settings

## Goal

* Update the repo to be deployed to npm
  * https://www.npmjs.com/package/@svkeg/re-theme
* Use the `@svkeg` organization
* Updated the dependencies to use the `@svkeg` org
  * See https://www.npmjs.com/search?q=%40svkeg

## Updates

* `.npmrc`
  * Removed
* `package.json`
  * Updated to be deployed to npm
* `container/docker-compose.yml`
  * Fixed the paths to tap 
* `scripts/autoLink.js`
  * script to autoLink the retheme as a tap to the keg-cli
* `container/mutagen.yml`
  * Moved `actions` to the values.yml file
  * This breaks them for now, but the keg-cli is in the process of being updated
* `...all other files`
  * Updated to use `@svkeg/jsutils`

## Testing

* Pull down branch
  * `cd /path/to/re-theme && keg pr 30`
* Install all deps
  * `yarn install`
  * Ensure all deps install properly. There should be no errors
* Run tests
  * `yarn test`
* Ensure all tests pass
* Use the keg-cli to start the container
  * Switch to this branch on the keg-cli => `keg-public-key-docker-updates`
  * Make sure retheme is linked as a tap => `cd /path/to/re-theme && keg tap link retheme`
  * Run retheme start command => `keg retheme start`
    * For now this will only start the container
  * Run the command again to actually start the server => `keg retheme start`
  * Ensure everything works as expected
